pom.xml

```diff
@@ -79,18 +86,36 @@
 	</dependencies>
 	<build>
 		<plugins>
-			<!-- <plugin> <groupId>io.openapitools.swagger</groupId> <artifactId>swagger-maven-plugin</artifactId> 
-				<configuration> <resourcePackages> <resourcePackage>io.openapitools.swagger.example</resourcePackage> 
-				<resourcePackage>io.openapitools.swagger.example.alternate</resourcePackage> 
-				</resourcePackages> <outputDirectory>${basedir}/target/</outputDirectory> 
-				<outputFilename>swagger</outputFilename> <outputFormats>JSON,YAML</outputFormats> 
-				<prettyPrint>true</prettyPrint> </configuration> <executions> <execution> 
-				<goals> <goal>generate</goal> </goals> </execution> </executions> </plugin> 
-				<plugin> <groupId>org.apache.maven.plugins</groupId> <artifactId>maven-compiler-plugin</artifactId> 
-				<version>3.8.1</version> <configuration> <source>1.8</source> <target>1.8</target> 
-				</configuration> </plugin> <plugin> <groupId>org.apache.maven.plugins</groupId> 
-				<artifactId>maven-javadoc-plugin</artifactId> <version>3.0.0</version> <configuration> 
-				<source>1.8</source> <target>1.8</target> </configuration> </plugin> -->
+			<plugin>
+				<groupId>io.openapitools.swagger</groupId>
+				<artifactId>swagger-maven-plugin</artifactId>
+				<version>2.1.5</version>
+				<configuration>
+					<resourcePackages>
+						<resourcePackage>com.carmen.app.boundary</resourcePackage>
+					</resourcePackages>
+					<outputDirectory>${basedir}/target/</outputDirectory>
+					<outputFilename>swagger</outputFilename>
+					<outputFormats>JSON,YAML</outputFormats>
+					<prettyPrint>true</prettyPrint>
+				</configuration>
+				<executions>
+					<execution>
+						<goals>
+							<goal>generate</goal>
+						</goals>
+					</execution>
+				</executions>
+			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-war-plugin</artifactId>
+				<version>3.3.1</version>
+				<configuration>
+					<outputDirectory>C:\Program Files\payara5\glassfish\domains\domain1\autodeploy</outputDirectory>
```

Comment:
Should not use an absolute path, it would not work for other members of the team. Check https://cwiki.apache.org/confluence/display/MAVEN/Maven+Properties+Guide?showComments=true&showCommentArea=true